sriovnet_switchdev_test.go

```diff
@@ -83,9 +151,7 @@ func TestGetUplinkRepresentorErrorEmptySwID(t *testing.T) {
 func TestGetUplinkRepresentorErrorMissingUplink(t *testing.T) {
 	var testErr error
 	vfPciAddress := "0000:03:00.4"
-	teardown := setupUplinkRepresentorEnv(t, vfPciAddress)
 	expectedError := fmt.Sprintf("failed to lookup %s", vfPciAddress)
-	defer teardown()
 	path := filepath.Join(PciSysDir, vfPciAddress)
 	testErr = utilfs.Fs.RemoveAll(path)
```

Comment:
I think this should be removed as well together with the defer below
as we are not creating anything, so no need to delete paths

Reply:
removed